diff --git a/node_modules/@types/relay-runtime/index.d.ts b/node_modules/@types/relay-runtime/index.d.ts
index 18ff05e..1ab09b6 100644
--- a/node_modules/@types/relay-runtime/index.d.ts
+++ b/node_modules/@types/relay-runtime/index.d.ts
@@ -193,12 +193,20 @@ export { createRelayNetworkLogger } from './lib/network/createRelayNetworkLogger
 export { deepFreeze } from './lib/util/deepFreeze';
 
 // These match the output of relay-compiler-language-typescript.
-export interface _RefType<T> {
-    ' $refType': T;
+export interface _RefType<T extends string> {
+  " $refType": T;
 }
-export interface _FragmentRefs<T> {
-    ' $fragmentRefs': T;
+
+export type FragmentRefs<Refs extends string> = {
+  [ref in Refs]: true
+}
+
+export interface _FragmentRefs<Refs extends string> {
+  " $fragmentRefs": FragmentRefs<Refs>
 }
 
 // This is a utility type for converting from a data type to a fragment reference that will resolve to that data type.
-export type FragmentRef<T> = T extends _RefType<infer U> ? _FragmentRefs<U> : never;
+export type FragmentRef<T> = T extends _RefType<infer U>
+  ? _FragmentRefs<U>
+  : never;
+
